scipy/optimize/_qap.py

```diff
@@ -0,0 +1,418 @@
+import numpy as np
+import math
+from . import linear_sum_assignment, minimize_scalar, OptimizeResult
+
+
+def quadratic_assignment(
+    cost_matrix,
+    dist_matrix,
+    seed=None,
+    maximize=False,
+    n_init=1,
+    init_method="barycenter",
+    max_iter=30,
+    shuffle_input=True,
+    eps=0.05,
+):
+    """
+    Solve the quadratic assignment problem
+
+    This class solves the Quadratic Assignment Problem (QAP) and the
+    Graph Matching Problem through an implementation of the Fast
+    Approximate QAP Algorithm (FAQ) (these two problems are the same up
+    to a sign change) [1]_.
+
+    This algorithm can be thought of as finding an alignment of the
+    vertices of two graphs which minimizes the number of induced edge
+    disagreements, or, in the case of weighted graphs, the sum of squared
+    differences of edge weight disagreements. The option to add seeds
+    (known vertex correspondence between some nodes) is also available
+    [2]_.
+
+    Note that the quadratic assignment problem is NP-hard, is not
+    known to be solvable in polynomial time, and is computationally
+    intractable. Therefore, the results given are approximations,
+    not guaranteed to be exact solutions.
+
+
+    Parameters
+    ----------
+    cost_matrix : 2d-array, square, non-negative
+        A square adjacency matrix
+
+    dist_matrix : 2d-array, square, non-negative
+        A square adjacency matrix
+
+    seed : 2d-array, optional, (default = None)
+        Allows the user apply a seed, fixing part of the matching between
+        the two adjacency matrices.
+        For column 1, each entry is an index of a node in `cost_matrix`.
+        For column 2, each entry is an index of a node in `dist_matrix`.
+        The elements of ``seed[:, 0]`` and ``seed[:, 1]`` are vertices
+        which are known to be matched, that is, `seed[i, 0]` is matched to
```

Comment:
I have made a few comments about backticks before. These need to be extended to the rest of the document; I won't comment on every occurrence of the issue. See how this renders?
![image](https://user-images.githubusercontent.com/6570539/82103999-0e8e7a00-96ca-11ea-96ff-cbf47e9cd6c6.png)

Please review the rendered documentation:
![image](https://user-images.githubusercontent.com/6570539/82104025-1fd78680-96ca-11ea-87bf-01c22eee146d.png)
after making changes.

Please review the